ecosystem/typescript/sdk/README.md

```diff
@@ -57,27 +37,37 @@ sudo apt-get install nodejs yarn
 ```
 
 ### Generating Types
+
```

Comment:
lots of new whitespace, does that show up in the actual pretty rendered version?

Reply:
it doesn't, no